src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -940,6 +940,20 @@ internal string GetPropertyRawText()
         /// <exception cref="ObjectDisposedException">
         ///   The parent <see cref="JsonDocument"/> has been disposed.
         /// </exception>
+        public void WriteAsProperty(string propertyName, Utf8JsonWriter writer)
+            => WriteAsProperty(propertyName.AsSpan(), writer);
```

Comment:
Is the intent to let `null` succeed?

Reply:
Yes. null == "" (consistent with how the writer works today).

We need to agree on the semantics as part of https://github.com/dotnet/corefx/issues/34632, but I wanted to keep it as is until then.